The failed pipelines failed at this stage:



26269
================== log:
$> /home/a-m/azzaea/software/nextflow/19.04.1 run strongScaling.nf -profile cluster --ntasks=32768 --duration=0
:
:
May-28 23:57:28.614 [Task submitter] DEBUG nextflow.processor.TaskRun - Unable to dump error of process 'strongScalability (process_26269)' -- Cause: java.nio.file.NoSuchFileException: /igbgroup/a-m/azzaea/scalability-tst/nf/work/ae/86ede4d01f05df09957b811f045cc1/.command.log
May-28 23:57:28.616 [Task submitter] ERROR nextflow.processor.TaskProcessor - Error executing process > 'strongScalability (process_26269)'

Caused by:
  Failed to submit process to grid scheduler for execution

Command executed:

  sbatch .command.run

Command exit status:
  1

Command output:
  sbatch: error: Batch job submission failed: Unable to contact slurm controller (connect failure)

Work dir:
  /igbgroup/a-m/azzaea/scalability-tst/nf/work/ae/86ede4d01f05df09957b811f045cc1

=================

70582
================== log:
$> /home/a-m/azzaea/software/nextflow/19.04.1 run strongScaling.nf -profile cluster --ntasks=131072 --duration=0
:
:
May-29 11:57:25.480 [Task submitter] DEBUG nextflow.processor.TaskRun - Unable to dump error of process 'strongScalability (process_70582)' -- Cause: java.nio.file.NoSuchFileException: /igbgroup/a-m/azzaea/scalability-tst/nf/work/35/c6b7a9997ea99b89c7e77de908238c/.command.log
May-29 11:57:25.482 [Task submitter] ERROR nextflow.processor.TaskProcessor - Error executing process > 'strongScalability (process_70582)'

Caused by:
  Failed to submit process to grid scheduler for execution

Command executed:

  sbatch .command.run

Command exit status:
  1

Command output:
  sbatch: error: Batch job submission failed: Unable to contact slurm controller (connect failure)

Work dir:
  /igbgroup/a-m/azzaea/scalability-tst/nf/work/35/c6b7a9997ea99b89c7e77de908238c


=================

